IBX-8686: Ibexa Commerce Product Guide added in Developer Documentation (#2460)
ibexa/documentation-developer: docs/ibexa_products/ibexa_commerce.md

---
title: Ibexa Commerce
description: Discover Ibexa Commerce and its essential features, capabilities and benefits.
month_change: true
---

# Ibexa DXP Commerce edition product guide

## What is [[= product_name_com =]]

[[= product_name_com =]] is the most powerful edition offered by [[= product_name_base =]].

It assists you in managing each aspect of your customers' journey by combining content management, customization, and commerce functions into a single, dedicated solution.

> [[= product_name_com =]] offers a streamlined, unified platform where you can personalize each component of the online shopping experience.

Reviewer: ```suggestion
[[= product_name_com =]] offers a streamlined, unified platform where you can personalize each aspect of the online shopping experience.
```
Preferential, but "component" ssounds material, while aspect can also include emotions and sensations.

You can completely revamp your online stores and give your consumers exceptional purchasing experiences, from first contact to post-purchase support.

![Ibexa Commerce](ibexa_commerce.png)

## Availability

> To start using [[= product_name_com =]] you need to purchase a product license.

Reviewer: ```suggestion
To start using [[= product_name_com =]], you need to purchase a product license.
```

For more information, see [[[= product_name_com =]] license pricing](https://www.ibexa.co/products/pricing?tab=3).
You can also [contact us](https://www.ibexa.co/about-ibexa/contact-us) or [one of our partners](https://www.ibexa.co/partners).

## How it works

### Technical backstage

> With an active lincese, you can start the [installation process](install_ibexa_dxp.md) that uses the Composer.

Reviewer 2: ```suggestion
With an active license, you can start the [installation process](install_ibexa_dxp.md) that uses the Composer.
```

[[= product_name_com =]] is based on [Symfony]([[= symfony_doc =]]).
With a help of documentation and trainings, any developer familiar with Symfony or even PHP alone can learn how to use available extension points and extend the platform.

Version control systems and environment variables allow you to deploy your extensions and settings on several environments, such as [Ibexa Cloud](ibexa_cloud_guide.md).

[[= product_name_com =]] is built on [[[= product_name_exp =]]](experience.md) and includes all bundles, APIs, and features that come with both [Headless](headless.md#core-features) and [Experience](experience.md#core-features) editions.

## Capabilities and benefits

With [[= product_name_com =]] you can focus on accelerating your transformation into a fully-fledged eCommerce.
It comes with all the necessary tools: customized catalogs, integration with CDP, personalized checkout workflows, payment gateway integration, transactional emails, and more.

### Core features

[[= product_name_com =]] includes all the features you need to launch your online store and reduce the time it takes to go live.

#### Order management

> With the advanced [Order management]([[= user_doc =]]/commerce/order_management/order_management/) tools, you can you can manage orders with ease.

Reviewer: ```suggestion
With the advanced [Order management]([[= user_doc =]]/commerce/order_management/order_management/) tools, you can manage orders with ease.
```

Depending on your permissions, you can search for orders, review their details and updates, track completion status, and cancel orders that are created when store customers purchase products.
When searching for orders, you can use filters to save time and reduce effort.
Order management is strongly connected with other components of the Commerce offering, such as [Cart](cart.md) and [Checkout](checkout.md), so users can speed up the process by uploading an order list or repeating previous transactions.

![Order management](order_management.png)

#### Payment management

> The [Payment](payment.md) component allows users to search for payment methods and payments, create new and manage existing payments and payment methods, and filter search results. Users can also enable or disable payment methods, change payment details, and cancel payments, depending on their role.

Reviewer: ```suggestion
The [Payment](payment.md) component allows users to search for payment methods and payments, create new and manage existing payments and payment methods, and filter search results.
Users can also enable or disable payment methods, change payment details, and cancel payments, depending on their role.
```

![Payment management](payment_management.png)

#### Shipping management

> With the [Shipping](shipping_management.md) component users can create and manage shipments, search for shipments, filter search results, and define and manage various shipping methods. Depending on their role, users can also enable or disable shipping methods, change status of shipments, and cancel shipments.

Reviewer: ```suggestion
With the [Shipping](shipping_management.md) component users can create and manage shipments, search for shipments, filter search results, and define and manage various shipping methods.
Depending on their role, users can also enable or disable shipping methods, change status of shipments, and cancel shipments.
```

> With the [Shipping](shipping_management.md) component ussers can create and manage shipments, search for shipments, filter search results, and define and manage various shipping methods. Depending on their role, users can also enable or disable shipping methods, change status of shipments, and cancel shipments.

Reviewer 2: ```suggestion
With the [Shipping](shipping_management.md) component users can create and manage shipments, search for shipments, filter search results, and define and manage various shipping methods. Depending on their role, users can also enable or disable shipping methods, change status of shipments, and cancel shipments.
```

![Shipping management](shipping_management.png)

#### Storefront

> The [Storefront](storefront.md) package includes a starter kit for developers. It is a foundational set of components that developers can customize and extend to create their own web store implementations.

Reviewer: ```suggestion
The [Storefront](storefront.md) package includes a starter kit for developers.
It is a foundational set of components that developers can customize and extend to create their own web store implementations.
```

> It contains default UI components and widgets that can be modified to [create customized web store](customize_storefront_layout.md).

Reviewer: ```suggestion
It contains default UI components and widgets that can be modified to [create a customized web store](customize_storefront_layout.md).
```

#### Relevant faceted search

> Search becomes crucial, when your product catalog is extensive.

Reviewer: ```suggestion
Search becomes crucial when your product catalog is extensive.
```

Products can be sorted according to a variety of criteria using faceted search.

> The value it brings makes it a crucial component in merchandizing. You can set up your search engine using [[= product_name_com =]] to help clients find what they're looking for more easily, which could result in more purchases.

Reviewer: ```suggestion
The value it brings makes it a vital component in merchandizing.
You can set up your search engine with [[= product_name_com =]] to help clients find what they're looking for more easily, which could result in more purchases.
```
Preferential, but you've used "crucial" two sentences prior.

#### Catalog management

[[= product_name_com =]] gives you the ability to manage your product repository - [PIM](pim_guide.md), and construct an infinite number of product catalogs, each with unique prices, to further customize the experience for your customers.

#### Transactional emails

> Commerce allows you to send transactional emails - messages that [[= product_name_base =]] can send through [Actito](https://www.actito.com/en-BE/) gateway to your end-users. These emails include notifications about changes in the status of various actions taken in relation to your commerce presence.

Reviewer: ```suggestion
Commerce allows you to send transactional emails - messages that [[= product_name_base =]] can send through the [Actito](https://www.actito.com/en-BE/) gateway to your end-users.
These emails include notifications about changes in the status of various actions taken in relation to your commerce presence.
```

> Commerce allows you to send transactional emails - messages that [[= product_name_base =]] can send through [Actito](https://www.actito.com/en-BE/) gateway to your end-users. These emails includes notifications about changes in the status of various actions taken in relation to your commerce presence.

Reviewer 2: ```suggestion
Commerce allows you to send transactional emails - messages that [[= product_name_base =]] can send through [Actito](https://www.actito.com/en-BE/) gateway to your end-users. These emails include notifications about changes in the status of various actions taken in relation to your commerce presence.
```

With this feature you can also [create email campaigns](transactional_emails.md##create-email-campaigns) to engage users and increase sales.

### Use cases

#### Create personalized shipping experience

Use [Ibexa Personalization]([[= user_doc =]]/personalization/use_cases/#ecommerce) to transform your online stores and give your consumers great buying experiences, from initial contact to post-purchase support.
No matter how complicated your product or sales process are, you can present your offer in an approachable way.

> Creating engaging and personalized shopping experiences with targeted offers and recommendations, helps you to boost sales.

Reviewer: ```suggestion
Creating engaging and personalized shopping experiences with targeted offers and recommendations helps you boost sales.
```

Within eCommerce, product recommendations can assist users in finding the exact item that meets their needs.
Recommendations can be used to propose related, alternative, or complimentary products to users who are unsure what to buy.

#### Use effective merchandising

Merchandising assists in keeping the consistency of the brand and providing customized product recommendations with captivating visuals and powerful search features. 
You can engage your customers with eye-catching graphics and information.

> The search engine makes it easy to find what you're looking for by providing quick and easy access to the product catalogs.

Reviewer: ```suggestion
The search engine makes it easy to find what they're looking for by providing quick and easy access to the product catalogs.
```
?

> The customer experience takes an important step further by facilitating financial transactions through the use of powerful, individualized product suggestions provided by [[= product_name_com =]] and unique pricing for various customer groups.

Reviewer: further -> forward ?

Author: Both correct, but 'forward' sounds better.

#### Launch consumer-facing web stores

[[= product_name_com =]] comes with all the features you need to launch and manage your web store, like storefront starter kit, real-time cart management, stock inventory, catalog management, and more.

> This edition is designed for complex enterprises and is fully customizable. It enables you to design flawless sales experiences regardless of the complexity of your business model.

Reviewer: ```suggestion
This edition is designed for complex enterprises and is fully customizable.
It enables you to design flawless sales experiences regardless of the complexity of your business model.
```

You can create online store that really fits your needs.

#### Increase B2B sales

> [[= product_name_com =]] contains the best B2B features to help you speed your digital transformation, such as corporate account management, tailored catalogs, customized workflows, effortless reordering, custom pricing, and more.

Reviewer: ```suggestion
[[= product_name_com =]] contains the best B2B features to help you speed up your digital transformation, such as corporate account management, tailored catalogs, customized workflows, effortless reordering, custom pricing, and more.
```

> #### Automate business processess

Reviewer 2: ```suggestion
#### Automate business processes
```

> With [[= product_name_com =]] automation process becomes easier, what's essential if your company wants to do more with less effort. 

Reviewer: ```suggestion
With [[= product_name_com =]], the automation process becomes easier, which is essential if your company wants to do more with less effort. 
```

It comes with all capabilities, including order and inventory management, customer data and custom pricing, which are needed to achieve it.
You can integrate with over 1,300 standard apps, including your CRM, ERP, PIM and DAM systems, and build custom connectors.

> Also, you can use ready-to-use pre-designed templates.

Reviewer: ```suggestion
Also, you can use the ready-to-use, pre-designed templates.
```

---
title: Ibexa Commerce
description: Discover Ibexa Commerce and its essential features, capabilities and benefits.
month_change: true
---

# Ibexa DXP Commerce edition product guide

## What is [[= product_name_com =]]

> [[= product_name_com =]] is one of the edition offered by [[= product_name =]].

Reviewer: I like the latter

This edition assists you in managing each aspect of the customer journey by combining content, customization, and commerce functions into a single, dedicated interface.
[[= product_name_com =]] offers companies a streamlined, unified platform where they can personalize each component of the online shopping experience.

> You can completely revamp your online stores with and give your consumers exceptional purchasing experiences, from first contact to post-purchase support.

Author: I prefer to use "you" to address the user directly. I rewrote the entire section.

![Ibexa Commerce](ibexa_commerce.png)

## Availability

To start using [[= product_name_com =]] you need to purchase a product license.
For more information, see [[[= product_name_com =]] license pricing](https://www.ibexa.co/products/pricing?tab=3).
You can also [contact us](https://www.ibexa.co/about-ibexa/contact-us) or [one of our partners](https://www.ibexa.co/partners).

## How it works

### Technical backstage

> With an active lincese, you can start [installation process](install_ibexa_dxp.md) that uses the Composer.

Reviewer: 50/50
"that uses" is a preferred replacement of "using" (we'd rather not use gerunds)

> [[= product_name_com =]] is based on [Symfony]([[= symfony_doc =]]). With a help of documentation and trainings, any developer familiar with Symfony or simply PHP may learn how to use available extension points and extend the platform.

Author: This part was copied from another Product Guides (Headless basically) to have it consistent.
I'll change it here and there (in another PR).

> Version control systems and environment variables allow you to deploy your extensions and settings on several environments, such as [Ibexa Cloud](ibexa_cloud_guide.md).

Author: Link will be added in the PR that focuses on reorganization and unification (https://issues.ibexa.co/browse/IBX-8590).

> [[= product_name_com =]] is built on [[[= product_name_headless =]]](headless.md) and includes all bundles, APIs, and [features that comes with Headless edition](headless.md#core-features) as well as [Experience features](experience.md#core-features).

Author: I've combined your suggestions.
[[= product_name_com =]] is built on [[[= product_name_ext =]]](experience.md) and includes all bundles, APIs, and features that come with both [Headless](headless.md#core-features) and [Experience](experience.md#core-features) editions.

## Capabilities and benefits

[[= product_name_com =]] includes the greatest capabilities to accelerate your digital transformation: Personalized workflows, corporate account management, customized catalogs, and more.

### Core features

[[= product_name_com =]] includes all the features you need to launch your online store and reduce the time it takes to go live.

> #### Order management

Reviewer 2: If we're mentioning order management - maybe we should mention Shipping and Payment management as well?

With the advanced [Order management]([[= user_doc =]]/commerce/order_management/order_management/) tools, you can easily manage your customized purchase process.
The Order management module allows users to review order details and updates, track order completion status, and cancel orders that are created when store customers purchase products, for a smooth purchasing process.
You can also assign and maintain rights so that only authorized users and make modifications.
When searching for orders you can use filters, so you save time and reduce effort.

![Order management](order_management.png)

> #### Merchandizing

Author: Regarding 1. In Product Guides, I believe, we don't need to follow any page structure/order. Anyway, it's not a problem, I can move it.
Regarding 2. Core features for Commerce, following ibexa.co, are: eCommerce, Merchandizing, and Order Management.
I also see Merchandizing as a core feature, not just use case.
<img width="733" alt="image" src="https://github.com/user-attachments/assets/55b65eb5-24b4-4d63-bf1b-64ee44651f1b">

Merchandizing assists in keeping the consistency of the brand and providing customized product recommendations with captivating visuals and powerful search features. 

> With this feature you can engage your customers with eye-catching graphics and information

Author: This is not a requirement.

> With this feature you can engage your customers with eye-catching graphics and information

Author: My comment was regarding breaks in the preview.

> With this feature you can engage your customers with eye-catching graphics and information

Reviewer 2: Ok, my comment was only about the missing divide between these two sentences - either adding dot or a line break is good enough for me.

Please note that Tomek also suggests adding a comma into this sentence

The search engine makes it easy to find what you're looking for by providing quick and easy access to the product catalogs.

> You can also use recommendations and campaigns to increase average order values and income.

Reviewer 2: 1) The whole section below is about recommendations, I wouldn't mix in in here as well
2) What are campaings? If I wanted to learn more about them - could we link to another part of the doc where they are described?
3) I'd mention the Storefront here - as a set of ready to use components that are included out of the box and that can be used to develop the websites faster (reducing time to market = 🏆 )

> #### eCommerce
> 
> With the [eCommerce feature]([[= user_doc =]]/personalization/use_cases/#ecommerce) you can transform your online stores and give your consumers great buying experiences, from initial contact to post-purchase support.

Reviewer 2: ```suggestion
#### Product recommendations

Use [Ibexa Personzalition]([[= user_doc =]]/personalization/use_cases/#ecommerce) to transform your online stores and give your consumers great buying experiences, from initial contact to post-purchase support.
```
I wouldn't use the name "eCommerce feature" - here you're describing the Personalization use case for eCommerce.

No matter how complicated your product or sales process are, you can present your offer in an approachable way.
Creating engaging and personalized shopping experiences with targeted offers and recommendations, helps you to boost sales.
Within eCommerce, product recommendations can assist users in finding the exact item that meets their needs.
Recommendations can be used to propose related, alternative, or complimentary products to users who are unsure what to buy.

![eCommerce](ecommerce.png)

> ### Use cases

Reviewer 2: I think all of these below (except for "eCommerce") are just features. For example: "Relevant faceted search" is not an use case, it's a feature you can use to achieve the usecase ("display products to the user").

I'd focus on two "high level" use cases:
- B2C: I want to sell my products directly to the Customers - how can Ibexa Commerce help me? With Site Factory, Page Builder, PIM, ..., faceted search, ..., connect, Personalization, (lots of options here!)
- B2B: I want to sell my products to other Companies -> how can Ibexa Commerce help me? Almost the same as above, but expanded additionaly with Customer Portals, Customer Groups - which are tailored at B2B use cases and building the long-term relations with the customers(you don't have a Sales Rep in B2C, but you do in B2B).

I really like the use cases we have in https://ez-systems-developer-documentation--2460.com.readthedocs.build/en/2460/ibexa_products/headless/#use-cases for Headless- it describes the real-life situations when someone might consider using Headless (and it even mentions when to use Ibexa Commerce - to finally become a store and sell online - so I'd expand on that).

Reviewer: Agree! Let's put Merchandising , consumer-facing web store and B2B the three use cases.

#### Custom eCommerce

[[= product_name_com =]] is designed for complex enterprises, so it enables you to design flawless sales experiences even if your business model has built-in limitations, like distribution, price, and so on. Thanks to this, [[= product_name_com =]] is fully customizable.
You can create online store that really fits your needs.

#### Immersive shopping experiences

The customer experience takes an important step further by facilitating financial transactions through the use of powerful, individualized product suggestions provided by [[= product_name_com =]] and unique pricing for various customer groups.

#### Relevant faceted search

> Search becomes crucial, when your product catalog is extensive.

Reviewer: Faceted search is a PIM feature.

> Search becomes crucial, when your product catalog is extensive.

Author: It's presented as a use case in Merchadizing. For me makes sense.
